test/modules/layers/path-layer/path-layer-vertex.spec.js

```diff
@@ -0,0 +1,41 @@
+import test from 'tape-catch';
+
+// import {COORDINATE_SYSTEM, Viewport, WebMercatorViewport} from 'deck.gl';
+import {gl} from '@deck.gl/test-utils';
+import {Transform, Buffer} from 'luma.gl';
+import VS from '../../../../modules/layers/src/path-layer/path-layer-vertex.glsl';
```

Comment:
Should import from `@deck.gl/layers/path-layer/...` and rely on module-alias to resolve the path.

Using relative path may break dist test.